strapi/strapi
# fix: admin users email filter

What does it do?
This PR fixes an issue in the admin user section where filtering, sorting, or searching by the email field is not functioning properly. The email field is now included in the ADMIN_USER_ALLOWED_FIELDS, allowing for proper filtering and sorting.

Why is it needed?
The current restriction prevents users from efficiently managing admin accounts. Email is a critical identifier for admin users, and its absence in filtering and sorting operations can lead to significant usability issues.

How to test it?
1. Navigate to the Admin Users section.
2. Attempt to filter, sort, or search for users by email (e.g., enter a known admin user's email).
3. Verify that the operation completes successfully without any validation errors.
4. Ensure the results returned accurately reflect the expected admin users based on the email search.

https://github.com/user-attachments/assets/aff45ecb-21ee-4ffd-ba9d-bc1b3c1a9a10

Related issue(s)/PR(s)
fixes #21536 
fixes #21169 
fixes #21193
